Tasks/CloudFormationCreateOrUpdateStack/helpers/CreateOrUpdateStackTaskParameters.ts

```diff
@@ -15,13 +15,18 @@ export class TaskParameters extends AWSTaskParametersBase {
     public static readonly fileSource: string = 'file';
```

Comment:
1. Nit: Suggest to remove the `: string` annotations, as they prevent the TS compiler from inferring the more specific types `urlSource[: 'url'] = 'url'`, etc. The string literal types add extra type safety, and provide extra context when you mouse-over them in your favorite editor/IDE.
2. Rather than using `static readonly` class fields, consider exporting these as top-level constants:

  ```typescript
  export const urlSource = 'url'
  export const s3Source = 's3'
  // etc.
  ```

Reply:
when this module is refactored we can look into that